spec/coverage_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'simplecov'
+
+SimpleCov.minimum_coverage 54
+SimpleCov.start 'rails' do
+  add_filter '/bin/'
+  add_filter '/config/'
+  add_filter '/jobs/application_job.rb'
+  add_filter '/schemas/'
+  add_filter '/lib/generators'
+  add_filter '/spec/'
+  add_filter '/vendor/'
```

Comment:
I wonder if we need to add more entries here, like:
/public
/swagger
/script
/log
/db

Reply:
@luisramos0 Added these above-mentioned directories. Ya, this is not yet covering the features spec hence showing LOCs less than what you mentioned here. Not sure how cucumber specs are integrated here so might be worth creating another PR for including feature specs to the coverage too.

> Nice, I think this is a good thing to have.
> 
> I just wonder what are those 17klocs...
> 
> I see at least ~40klocs:
> ~/dev/ofn (master) $ find app -name *.rb | xargs cat | wc -l
> 24490
> ~/dev/ofn (master) $ find lib -name *.rb | xargs cat | wc -l
> 6921
> ~/dev/ofn (master) $ find engines -name *.rb | xargs cat | wc -l
> 9220
> 
> Also, 54% looks really low, I wonder if this includes feature specs (a lot of our tests are feature specs)?

Reply:
nice, thanks!